bigquery/tables.js

```diff
@@ -80,6 +80,19 @@ function listTables (options, callback) {
 }
 // [END list_tables]
 
+function browseRows (dataset, table, callback) {
+  var bigquery = BigQuery();
+  var tableObj = bigquery.dataset(dataset).table(table);
+  tableObj.getRows(function (err, rows) {
```

Comment:
Add a space between `tableObj` and `table.getRows` for readability. Also, add the following above `tableObject.getRows`:

```
// See https://googlecloudplatform.github.io/google-cloud-node/#/docs/bigquery/latest/bigquery/table?method=getRows
```